CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Mattermost Mobile Apps Changelog
 
-## v1.1 Release (In Progress)
+## v1.1 Release
 
-- Planned Release Date: August 2017 
+- Release Date: August 2017 
```

Comment:
I think the last time we added the exact date once the AppStore approved the release